src/System.Net.Requests/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+// We have to run them with no parallelism because ServicePointManager.DefaultConnectionLimit is left as 2 in netfx and when they are running in parallel
+// it leaves too many connections open at once which causes tests to hang since they are not able to get a new HttpWebRequest connection opened.
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true, MaxParallelThreads = 1)]
```

Comment:
Would it make sense to have a netfx configuraiton just for the tests -- so we can put #if netfx around this and in the normal case it continues to run in parallel?

not sure if that's okay for a netstandard library but presumably it is. @joperezr